apps/mitt-konto/src/MittKonto/Main.purs

```diff
@@ -103,25 +103,26 @@ render self@{ state, setState } =
         [ foldMap alertView state.alert ]
     , classy DOM.div "mt4 mb4 clearfix"
         [ classy DOM.div "mitt-konto--main-container col-10 lg-col-7 mx-auto"
-            [ element Router.switch { children: [ paymentList, mittKonto ] } ]
+            [ element Router.switch { children: [ paymentListRoute, mittKontoRoute, noMatchRoute ] } ]
         ]
     , footerView
     ]
  where
-   mittKonto =
+   mittKontoRoute =
      element
        Router.route
          { exact: true
          , path: toNullable $ Just "/"
-         , render:
-             \_ -> classy DOM.div "mitt-konto--container clearfix"
-               [ foldMap loadingIndicator state.loading
-               , case state.loggedInUser of
-                   Just user -> userView self user
-                   Nothing   -> loginView self
-               ]
+         , render: \_ -> mittKontoView
```

Comment:
I guess we could use `const` here as well?

Reply:
Yes